inference-engine/tests/ie_test_utils/common_test_utils/data_utils.hpp

```diff
@@ -152,6 +152,20 @@ static void fill_data_roi(float *data, size_t size, const uint32_t range, const
     }
 }
 
+template<class T>
+void inline fill_data_random(T* pointer, std::size_t size, const uint32_t range = 10, int32_t start_from = 0, const int32_t k = 1, const int seed = 1) {
+    testing::internal::Random random(1);
```

Comment:
`random(seed)`? Or `seed` is not going to be used?

Reply:
Nice catch! I just copy this part of code from original function, but I guess it's an issue. @mikhail-treskin do you agree?

Reply:
Fixed